streaming/src/test/scala/org/apache/gearpump/streaming/StreamingTestUtil.scala

```diff
@@ -38,4 +50,62 @@ object StreamingTestUtil {
     executorId += 1
     miniCluster.launchActor(props).asInstanceOf[TestActorRef[AppMaster]]
   }
+
+  /**
+   * This function is a work-around for testing.
+   * Two ActorSystem parameters are used to simulate two differnent Workers
+   */
+  def createEchoForTaskActor(taskClass: String, taskConf: UserConfig, system1: ActorSystem, system2: ActorSystem): (ActorRef, TestProbe) = {
```

Comment:
This seems a quite complicated method,

Please add more comments,
About how it works.